.eslintignore

```diff
@@ -1 +1 @@
-video_xblock/static/js/player_state.js
+video_xblock/static/js/player-state.js
```

Comment:
Since we're here already. What effort is it going to take to remove this file from `.eslingignore`?